src/main/kotlin/racingcar/model/RaceManager.kt

```diff
@@ -0,0 +1,47 @@
+package racingcar.model
+
+class RaceManager {
```

Comment:
아래와 같이 생성자에서 초기화하면 요구사항 문제와 함께 해결될 듯 싶습니다!

```suggestion
class RaceManager(private val cars: List<Car>, private val raceCount: Int)
```